truck_inspection_client/config/parameters.yaml

```diff
@@ -0,0 +1,34 @@
+# Vehicle VIN number - 17 alphanumeric char
+vin_number: 1FUJGHDV0CLBP8834
```

Comment:
Please update to "1FUJGHDV0CLBP8896" for Red Truck. The one above was for White Truck by mistake.